central/reports/validation/validate.go

```diff
@@ -22,9 +22,9 @@ import (
 
 const (
 	// CustomEmailSubjectMaxLen is the maximum allowed length for custom email subject
-	CustomEmailSubjectMaxLen = 250
+	CustomEmailSubjectMaxLen = 256
 	// CustomEmailBodyMaxLen is the maximum allowed length for custom email body
-	CustomEmailBodyMaxLen = 1000
+	CustomEmailBodyMaxLen = 1500
```

Comment:
Introduce env var to control these and have both backend and frontend read from those env var, for reason below:
1. For the very reason you are doing this change.
2. It is likely that some customers will be dismayed by the default limits.